doc/source/authentication.rst

```diff
@@ -208,6 +208,28 @@ and obtain the confidential client credentials.
          userdata_params: {'state': 'state'}
          username_key: preferred_username
 
+Auth0
+^^^^^^^^^^^^^^
+
+Auth0 is a popular commercial provider of identity management. The jupyterhub helm does not support Auth0 as a
```

Comment:
...JupyterHub helm chart does not include Auth0 support by default. To use Auth0, ``extraEnv`` and ``extraConfig`` need to be configured as follows:

Reply:
Thanks, I merged what we both wrote.